joomla/joomla-cms
# Display max input vars (redo #7453)

Quote form @brianteeman see: https://github.com/joomla/joomla-cms/pull/7453#issue-95619872

> Since php 5.3.9 there is a php setting for the number of max_inut_vars with a default value of 1000
> 
> This pr just adds a test to display the value in com_admin&view=sysinfo
> 
> Knowing the value is quite important now especially on sites with a large number of menus, usergroups or modules as if the value for max_input_vars is not big enough then saving will fail silently or will be truncated.
#### What is this

This PR just displays the setting "Max Imput Vars" to the Backend PHP settings.
### How to test
- login to the backend
- Go to the Systeminformation (BE -> System -> Systeminformation)
- There is a Tab "PHP Settings"
- Confirm that bevor the patch there is no entry for "max input vars"
- apply the patch
- confirm that we now show the may input vars
#### Original Idea

Kudos goes to @brianteeman see: http://issues.joomla.org/tracker/joomla-cms/7453
